content/docs/command-reference/exp/save.md

```diff
@@ -14,21 +20,33 @@ usage: dvc exp save [-h] [-q | -v] [-f]
 
 ## Description
 
-Provides a way to save the current status of your <abbr>project</abbr> as an
-<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+Saves a snapshot of your <abbr>project</abbr> as an <abbr>experiment</abbr>,
+without polluting your Git repository with unnecessary commits, branches,
 directories, etc.
 
-Only files tracked by either Git or DVC are saved to the experiment. Use the
-`--include-untracked` (`-I`) option to explicitly include any untracked files in
-the experiment.
-
-`dvc exp save` creates <abbr>experiments</abbr> just like
-[`dvc exp run`](/doc/command-reference/exp/run), but does not require
-reproduction or setup of [pipeline stages](/doc/start/data-pipelines), making it
-possible to quickly start tracking, [comparing] and [persisting] experiments.
+This lets you start tracking, [comparing], [plotting], and [sharing] experiments
+quickly after making any project changes (e.g. retrain an ML model).
 
 [comparing]: /doc/user-guide/experiment-management/comparing-experiments
-[persisting]: /doc/user-guide/experiment-management/persisting-experiments
+[plotting]: /doc/user-guide/experiment-management/visualizing-plots
+[sharing]: /doc/user-guide/experiment-management/persisting-experiments
+
+<admon type="tip">
+
+Only files tracked by either Git or DVC are saved to the experiment unless you
+use `--include-untracked` (`-I`) on untracked files explicitly (see an
+[example](#examples) below).
+
+</admon>
+
+[Review] your experiments with `dvc exp show`. Successful ones can be [made
+persistent] by restoring them via `dvc exp branch` or `dvc exp apply` and
+committing them to the Git repo. Unnecessary ones can be [cleared] with
+`dvc exp gc`.
```

Comment:
```suggestion
`dvc exp remove`.
```

We are considering dropping `dvc exp gc` since it is more confusing and does almost the same thing as `dvc exp remove`.